extensions/sql-database-projects/package.nls.json

```diff
@@ -34,5 +34,7 @@
 	"sqlDatabaseProjects.Settings": "Database Projects",
 	"sqlDatabaseProjects.netCoreInstallLocation": "Full path to .NET Core SDK on the machine.",
 	"sqlDatabaseProjects.netCoreDoNotAsk": "Whether to prompt the user to install .NET Core when not detected.",
-	"sqlDatabaseProjects.welcome": "No database projects currently open.\n[New Project](command:sqlDatabaseProjects.new)\n[Open Project](command:sqlDatabaseProjects.open)\n[Create Project From Database](command:sqlDatabaseProjects.importDatabase)"
+	"sqlDatabaseProjects.welcome": "No database projects currently open.\n[New Project](command:sqlDatabaseProjects.new)\n[Open Project](command:sqlDatabaseProjects.open)\n[Create Project From Database](command:sqlDatabaseProjects.importDatabase)",
+
+	"sqlDatabaseProjects.insertSqlInputBinding":"Insert SQL Binding into Azure Function"
```

Comment:
nit : should it have a prefix like _Azure Function_ or _Sql Binding_ for easy find?

Reply:
sure I can make the category "Azure Functions" so it'll be like the other Azure function commands contributed by the Azure functions extension

Reply:
updated name for now. Will follow up with AF team on if it's ok to put it in the same category
![image](https://user-images.githubusercontent.com/31145923/128221902-30917992-8a9f-41fe-b7bc-6710965bf3c5.png)